vendor/bat-native-ads/src/bat/ads/internal/ad_pacing/ad_pacing_util.h

```diff
@@ -7,14 +7,15 @@
 #define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_AD_PACING_AD_PACING_UTIL_H_
 
 #include "base/rand_util.h"
+#include "bat/ads/internal/ad_pacing/ad_pacing_random_util.h"
 #include "bat/ads/internal/logging.h"
 
 namespace ads {
 
 template <typename T>
 bool ShouldPaceAd(const T& ad) {
-  const double rand = base::RandDouble();
-  if (rand <= ad.ptr) {
```

Comment:
Changed comparison to strict less, because when `ad.ptr == 0` then ad could be shown if `rand == 0`, but this shouldn't happen.